### ISSUE TITLE
Make default settings for gnt-cluster init configurable

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#988]
Originally reported of Google Code with ID 932.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

<b>What distribution are you using?</b>
all

<b>What steps will reproduce the problem?</b>
gnt-cluster init [...] my.cluster.tld

<b>What is the expected output? What do you see instead?</b>
Cluster is initialized with:
- --master-netdev=xen-br0
+ --master-netdev=br0                          // (as this is the default naming for the first bridge)
- --nic-parameters=link=xen-br0
+ --nic-parameters=link=br0
- --vg-name=xenvg
+ --vg-name=vg || --vg-name=lvm
- --hypervisor-parameters=kvm:kernel_path=/boot/vmlinuz-3-kvmU
+ --hypervisor-parameters=kvm:kernel_path=     // (as /boot/vmlinuz-3-kvmU is not to be found on any distribution known to me, unless you create it)
- --backend-parameters=memory=128M
+ --backend-parameters=memory=1G               // (as hardly an current OS runs with 128M of RAM)

<b>Please provide any additional information below.</b>

```
Originally added on 2014-08-28 14:51:45 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
In the present form, the issue is invalid. Let's not have
bikeshedding discussions on defaults.

Also, the default values for the kernel paths are already
configurable at configure time.

What we can do, is to allow the remaining default values 
(in particular the default for master-netdev) to
be overridden at configure time. In this way, you, or the
maintainer of your distribution, can change the defaults to
fit with your system/distribution.

However, I see no reason to change the defaults for the defaults.
(This opinion is also shared by Petr.)

Rewording the summary line accordingly.
```
Originally added on 2014-08-28 15:29:36 +0000 UTC.

Added Labels: Type-Enhancement Priority-Low
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
oke, that looks fine to me.

so I should then talk to Apollon because of the defaults on Debian.

```
Originally added on 2014-08-28 16:17:14 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-08-28 16:30:48 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 2dbccc3bc572eaaadbe0e7ea10c85f8c8f2297ac
Author: Klaus Aehlig <aehlig@google.com>
Date:   Thu Aug 28 18:42:35 2014 +0200

    Use configure-provided value for the default bridge
    
    ...instead of hard-coding the value "xen-br0".
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

commit e3c309e239c287731df36b3842435dcc5728c72e
Author: Klaus Aehlig <aehlig@google.com>
Date:   Thu Aug 28 18:39:19 2014 +0200

    Add --with-default-bridge configure option
    
    This option will provide the make variable DEFAULT_BRIDGE,
    defaulting to the current default "xen-br0".
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2014-08-29 08:29:05 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 8b95604483dac15d7d49808a460c5062be1a00fe
Author: Klaus Aehlig <aehlig@google.com>
Date:   Fri Aug 29 14:55:01 2014 +0200

    Use configure-provided value for default volume group
    
    ...instead of hard coding "xenvg".
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

commit d14c8a973863274f7d58502b4b702a21985c2d7d
Author: Klaus Aehlig <aehlig@google.com>
Date:   Fri Aug 29 14:45:02 2014 +0200

    Add --with-default-vg configure option
    
    This option will provide the make variable DEFAULT_VG,
    defaulting to the current default "xenvg".
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2014-08-29 13:22:11 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:02 +0000 UTC.

Changed State: Released
